modules/core/src/main/scala/org/tessellation/domain/trust/storage/OrdinalTrustMap.scala

```diff
@@ -0,0 +1,34 @@
+package org.tessellation.domain.trust.storage
+
+import cats.Order
+import cats.syntax.contravariant._
+
+import org.tessellation.cli.method.ordinalTrustUpdateInterval
+import org.tessellation.schema.SnapshotOrdinal
+
+import derevo.cats.eqv
+import derevo.circe.magnolia.encoder
+import derevo.derive
+
+@derive(eqv, encoder)
+case class OrdinalTrustMap(
+  ordinal: SnapshotOrdinal,
+  peerLabels: PublicTrustMap,
+  trust: TrustMap
+)
+
+object OrdinalTrustMap {
+
+  implicit val order: Order[OrdinalTrustMap] = Order[SnapshotOrdinal].contramap(_.ordinal)
+
+  def empty: OrdinalTrustMap = OrdinalTrustMap(SnapshotOrdinal.MinValue, PublicTrustMap.empty, TrustMap.empty)
+
+  def initial: OrdinalTrustMap = OrdinalTrustMap(SnapshotOrdinal(ordinalTrustUpdateInterval), PublicTrustMap.empty, TrustMap.empty)
```

Comment:
What is the difference between `empty` and `initial`? Why do you need both?

Reply:
Should the `initial` be depending on the value defined in cli directly? Rather than take it as a parameter? If it's really hardcoded for every configuration maybe it should be defined in the `OrdinalTrustMap` object rather than configuration? 🤔

Reply:
`current` is initialized to `empty`. `next` is initialized to `initial`. The difference between the two is `next` is updated every `ordinalTrustUpdateInterval`. `next.ordinal` is the ordinal when `next` is updated again. So, `initial` initializes an `OrdinalTrustMap` to `ordinalTrustUpdateInterval`, the first time `next` is updated.